### DISCUSSION TITLE
How to configure User Defined Variables

### DISCUSSION THREAD

**Discussion by ghost** [qualersoft/jmeter-gradle-plugin#129] [Q&A]
Hi everyone, 

I had an issue and was able to find a solution, thank you for an awesome plugin! Please mention in the documentation, that all the typical JMeter flags can be applied for tasks. I know this should be obvious, but it wasn't for me :P 

Best regards,
Sewa

**Comment by mathze** [comment]
Hi Sewa,

Glad to hear that you find the plugin useful.
Regarding your suggestion: Can you be more specific about what you mean by "all the typical JMeter flags can be applied for tasks."
Do you mean the arguments exposed by the plugin itself via the [`jmeter`-extension](https://github.com/qualersoft/jmeter-gradle-plugin#customization) or do you mean jmeter-arguments not provided by the plugin?

Cheers,
Mathze
